document-generation/igce-document.ts

```diff
@@ -262,7 +262,12 @@ export async function generateIGCEDocument(
   const contractingShopFeePct = payload.contractingShop.fee * 0.01;
   if (contractingShopFeePct) {
     const contractingShopFeeCell = summarySheet.getCell("C27");
-    contractingShopFeeCell.value = { formula: `=(K24 + K25) * ${contractingShopFeePct}`, date1904: false };
+    if (payload.contractingShop.name !== "DITCO") {
+      contractingShopFeeCell.value = { formula: `=(K24 + K25) * ${contractingShopFeePct}`, date1904: false };
+    }
+    if (payload.contractingShop.name === "DITCO") {
+      contractingShopFeeCell.value = { formula: `=K24 * ${contractingShopFeePct}`, date1904: false };
+    }
```

Comment:
Little bit of a nitpick but from a code structure standpoint this could be simplified into
`if (payload.contractingShop.name === "DITCO") {
      contractingShopFeeCell.value = { formula: `=K24 * ${contractingShopFeePct}`, date1904: false };
    } else {
      contractingShopFeeCell.value = { formula: `=(K24 + K25) * ${contractingShopFeePct}`, date1904: false };
    }`

Reply:
Whew that code did not come out pretty in my comment. Basically my point is that for efficiency and readability you can replace the two `if` blocks with an if/else